docs/ref/forms/fields.txt

```diff
@@ -369,6 +369,13 @@ For each field, we describe the default widget used if you don't specify
     If provided, these arguments ensure that the string is at most or at least
     the given length.
 
+.. versionadded:: 1.8
```

Comment:
If I understood properly, `versionadded` should be at the same level as what's in it, except when we want to remove the whole block 2 versions later.